src/backends/dlib/dliblib.cc

```diff
@@ -51,11 +51,24 @@ namespace dd {
             _net_type = ad.get("model_type").get<std::string>();
         }
 
-        if (_net_type.empty() || (_net_type != "obj_detector" && _net_type != "face_detector")) {
-            throw MLLibBadParamException("Must specify model type (obj_detector or face_detector)");
+        if (_net_type.empty() ||
+            (_net_type != "obj_detector" && _net_type != "face_detector" && _net_type != "face_feature_extractor")) {
+            throw MLLibBadParamException("Must specify model type (obj_detector, face_detector, or face_feature_extractor)");
+        }
+
+        if (ad.has("shape_predictor")) {
```

Comment:
I would prefer  if (ad.has("shape_predictor") && ad.get("shape_predictor").get<bool>())   just in case someone extra cautious put "shape_predictor" : false      :)

Reply:
👍